### PR TITLE
CLOUDP-264043: Integration tests for plugin install command

### PR DESCRIPTION
**Pull request by stefan4h**
## Proposed changes

- Add integration tests for `plugin install`

_Jira ticket:_ CLOUDP-264043

### CODE REVIEW

File: internal/cli/plugin/install.go [hunk 1]
```diff
@@ -35,8 +35,8 @@ type InstallOpts struct {
 }
 
 func (opts *InstallOpts) checkForDuplicatePlugins() error {
-	for _, plugin := range opts.plugins {
-		if plugin.Github != nil && plugin.Github.Equals(opts.githubRelease.owner, opts.githubRelease.name) {
+	for _, p := range opts.plugins {
```
Comment (jeroenvervaeke) [inline internal/cli/plugin/install.go:38]: Why did you rename this? `plugin` is more readable than `p`

File: internal/cli/plugin/plugin_asset.go [hunk 2]
```diff
@@ -226,7 +225,7 @@ func extractTarGz(src string, dest string) error {
 					return errCopyFileContentExtraction
 				}
 			}
-
+			outFile.Close()
```
Comment (jeroenvervaeke) [inline internal/cli/plugin/plugin_asset.go:228]: Wouldn't it be better to put this in a `defer` statement?
You will not close the file when you throw `errCopyFileContentExtraction`

**Review by jeroenvervaeke — CHANGES_REQUESTED** [review]

File: internal/cli/plugin/install.go (hunk 1 above)
Comment (stefan4h) [reply internal/cli/plugin/install.go:38]: I got an error from the linter saying that I cannot use "plugin" because it is the name of a package

File: internal/cli/plugin/plugin_asset.go (hunk 2 above)
Comment (stefan4h) [reply internal/cli/plugin/plugin_asset.go:228]: The linter threw me an error saying that I should not use defer in a loop

Comment (jeroenvervaeke) [reply internal/cli/plugin/plugin_asset.go:228]: Can you close the file before returning the error too?

**Review by jeroenvervaeke — CHANGES_REQUESTED** [review]

**Review by jeroenvervaeke — APPROVED** [review]
LGTM
